packages/mobile/src/identity/reducer.ts

```diff
@@ -238,6 +238,18 @@ export const reducer = (
           false
         ),
       }
+    case Actions.DELETE_USER_DATA:
```

Comment:
Calling this action `DELETE_USER_DATA` is a bit counter-intuitive, given that it's actually *saving* things.

Reply:
We're using `AccountActions.CLEAR_STORED_ACCOUNT` now, which could still be counter-intuitive but I think it's more expressive, so one could guess why we are saving data when clearing an account, what do you think?